src/api/sounds.ts

```diff
@@ -0,0 +1,38 @@
+import firebase from "firebase/app";
+
+import { SoundConfig, SoundConfigMap, SoundConfigSchema } from "types/sounds";
+
+import { withId } from "utils/id";
+import { itemsToObjectByIdReducer } from "utils/reducers";
+
+export const fetchSoundConfigs = async (): Promise<SoundConfigMap> => {
+  try {
+    const soundConfigsSnapshot = await firebase
+      .firestore()
+      .collection("sounds")
+      .withConverter(soundConfigConverter)
+      .get();
+
+    return soundConfigsSnapshot.docs
+      .map((docSnapshot) => withId(docSnapshot.data(), docSnapshot.id))
+      .reduce(itemsToObjectByIdReducer, {});
+  } catch (err) {
+    // TODO: properly log this with bugsnag or similar?
+    console.error("fetchSoundConfigs", err);
```

Comment:
[TODO] Log error with Bugsnag

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1269/commits/0d496de6156ad43094c293764036304f458d8477